.github/workflows/cabal.project.local.ci.MINGW64_NT-10.0-20348

```diff
@@ -0,0 +1,23 @@
+package cardano-crypto-praos
+  flags: -external-libsodium-vrf
+
+package HsOpenSSL
+  extra-include-dirs: D:/a/_temp/msys64/mingw64/include
+  extra-lib-dirs: D:/a/_temp/msys64/mingw64/lib
+  flags: +use-pkg-config
+
+package secp256k1-haskell
+  extra-include-dirs: D:/a/_temp/msys64/mingw64/include
+  extra-lib-dirs: D:/a/_temp/msys64/mingw64/lib
+
+package basement
+  extra-include-dirs: D:/a/_temp/msys64/mingw64/include
+  extra-lib-dirs: D:/a/_temp/msys64/mingw64/lib
+
+package cardano-crypto-class
+  extra-include-dirs: D:/a/_temp/msys64/mingw64/include
+  extra-lib-dirs: D:/a/_temp/msys64/mingw64/lib
+
+package cardano-crypto-praos
+  extra-include-dirs: D:/a/_temp/msys64/mingw64/include
+  extra-lib-dirs: D:/a/_temp/msys64/mingw64/lib
```

Comment:
And Haskell packages that fails to build because the system installation directory is missing will need an entry here.